sdk/quantum/quantum-jobs/package.json

```
@@ -6,79 +6,116 @@
   "dependencies": {
     "@azure/core-paging": "^1.1.1",
     "@azure/core-http": "^1.2.0",
-    "tslib": "^2.0.0"
+    "tslib": "^2.0.0",
+    "@opentelemetry/api": "^0.10.2"
   },
   "keywords": [
     "node",
     "azure",
     "typescript",
     "browser",
-    "isomorphic"
+    "isomorphic",
+    "cloud"
   ],
   "license": "MIT",
-  "main": "./dist/quantum-jobs.js",
-  "module": "./esm/index.js",
-  "types": "./esm/index.d.ts",
+  "main": "dist/index.js",
+  "module": "./dist-esm/src/index.js",
+  "types": "./types/latest/quantum-jobs.d.ts",
   "devDependencies": {
-    "typescript": "4.1.2",
-    "rollup": "^1.16.3",
     "rollup-plugin-node-resolve": "^3.4.0",
+    "mkdirp": "^1.0.4",
+    "@azure/dev-tool": "^1.0.0",
+    "@azure/eslint-plugin-azure-sdk": "^3.0.0",
+    "@azure/identity": "^1.1.0",
+    "@azure/test-utils-recorder": "^1.0.0",
+    "@microsoft/api-extractor": "7.7.11",
+    "@rollup/plugin-commonjs": "11.0.2",
+    "@rollup/plugin-inject": "^4.0.0",
+    "@rollup/plugin-json": "^4.0.0",
+    "@rollup/plugin-multi-entry": "^3.0.0",
+    "@rollup/plugin-node-resolve": "^8.0.0",
+    "@rollup/plugin-replace": "^2.2.0",
+    "@types/chai": "^4.1.6",
+    "@types/mocha": "^7.0.2",
+    "@types/node": "^8.0.0",
+    "@types/sinon": "^9.0.4",
+    "assert": "^1.4.1",
+    "chai": "^4.2.0",
+    "dotenv": "^8.2.0",
+    "eslint": "^7.15.0",
+    "esm": "^3.2.18",
+    "karma": "^5.1.0",
+    "karma-chrome-launcher": "^3.0.0",
+    "karma-coverage": "^2.0.0",
+    "karma-edge-launcher": "^0.4.2",
+    "karma-env-preprocessor": "^0.1.1",
+    "karma-firefox-launcher": "^1.1.0",
+    "karma-ie-launcher": "^1.0.0",
+    "karma-junit-reporter": "^2.0.1",
+    "karma-mocha": "^2.0.1",
+    "karma-mocha-reporter": "^2.2.5",
+    "karma-sourcemap-loader": "^0.3.8",
+    "mocha": "^7.1.1",
+    "mocha-junit-reporter": "^1.18.0",
+    "nock": "^12.0.3",
+    "nyc": "^14.0.0",
+    "prettier": "^1.16.4",
+    "rimraf": "^3.0.0",
+    "rollup": "^1.16.3",
+    "rollup-plugin-shim": "^1.0.0",
     "rollup-plugin-sourcemaps": "^0.4.2",
+    "rollup-plugin-terser": "^5.1.1",
+    "sinon": "^9.0.2",
+    "ts-node": "^8.3.0",
+    "typescript": "4.1.2",
     "uglify-js": "^3.4.9",
-    "@microsoft/api-extractor": "7.7.11",
-    "mkdirp": "^1.0.4"
+    "cross-env": "^7.0.2",
+    "typedoc": "0.15.2"
   },
   "homepage": "https://github.com/Azure/azure-sdk-for-js",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/Azure/azure-sdk-for-js.git"
-  },
+  "repository": "github:Azure/azure-sdk-for-js",
   "bugs": {
     "url": "https://github.com/Azure/azure-sdk-for-js/issues"
   },
   "files": [
+    "dist-esm/**/*.js",
+    "dist-esm/**/*.js.map",
+    "dist-esm/**/*.d.ts",
+    "dist-esm/**/*.d.ts.map",
     "dist/**/*.js",
     "dist/**/*.js.map",
     "dist/**/*.d.ts",
     "dist/**/*.d.ts.map",
```

- `.dts` files won't be generated in the `dist` or `dist-esm` folders since we have declared the "types" folder in the tsconfig. Hence, those entries can be removed.
- `dist-esm` would also contain the test files. They should be removed since these are the files to be published with the npm package.
```suggestion
    "dist/",
    "dist-esm/src/",
```